product_docs/docs/biganimal/release/overview/replication.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Faraway replicas"
+---
+
+You can create (faraway read) replicas of your *single node* and *high availability* clusters. In case of a failure, you can manually promote the faraway replica to a full-fledged BigAnimal cluster making it capable of accepting writes. See [Managing replicas](/biganimal/release/using_cluster/managing_replicas)
+
+## Advantages
+
+- **Disaster recovery**: You can create faraway replicas in different regions in your cloud. Deploying faraway replicas across regions can help you build a more solid disaster recovery (DR) plan.
+
+- **Improved read query performance**: If the faraway replica is set up in the same region, your application can benefit from an improved read query performance. For example, applications can read from an *us-west* replica the writes made to a cluster in the *us-east* region. 
+
+## Limitations
+
+- **Manual intervention**: Unlike standby replicas, BigAnimal does not automatically promote faraway replicas to the parent cluster in case of a failure. 
+    
+- **Replication lag**: Promoting a faraway replica to a BigAnimal cluster can result in loss of data. See [Log shipping](/biganimal/latest/using_cluster/04_backup_and_restore/#log-shipping).
+
+## Generic faraway clusters
```

Comment:
Can you add a description here explaining that 2 faraway replicas are attached to a 3-node HA cluster?

Reply:
Added